phptravels website/config.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Please remove all *.pyc files. They are not supposed to be pushed to the repo.

Reply:
Removed